### PR TITLE
POC: Porting Classic commands to BiDi in JS

### PR DESCRIPTION
**Pull request by pujagani**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

This is POC to demonstrate and act as a foundation for porting Classic commands to BiDi in JS

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [X] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by diemol — APPROVED** [review]

File: javascript/node/selenium-webdriver/firefox.js [hunk 1]
```diff
@@ -119,6 +119,7 @@ const zip = require('./io/zip')
 const { Browser, Capabilities } = require('./lib/capabilities')
 const { Zip } = require('./io/zip')
 const { getPath } = require('./common/driverFinder')
+const BiDiDelegator = require("./lib/bididelegator");
```
Comment (harsha509) [inline javascript/node/selenium-webdriver/firefox.js]: can we rename to `bidiDelegator`

File: javascript/node/selenium-webdriver/index.js [hunk 2]
```diff
@@ -44,6 +44,7 @@ const BrowsingContext = require('./bidi/browsingContext')
 const BrowsingContextInspector = require('./bidi/browsingContextInspector')
 const ScriptManager = require('./bidi/scriptManager')
 const NetworkInspector = require('./bidi/networkInspector')
+const {BiDiExecutor, getBiDiExecutorInstance} = require("./lib/bidiexecutor");
```
Comment (harsha509) [inline javascript/node/selenium-webdriver/index.js]: can we rename the filename to `bidiExecutor` as per js conventions

File: javascript/node/selenium-webdriver/lib/bidiexecutor.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    }
+
+    this.#currentContext = browsingContext
+
+    if (this.#commandHandlers.has(command.getName())) {
+      let f = this.#commandHandlers.get(command.getName())
+      let value = await f(this.#driver, command)
+
+      // Return value in the expected format
+      return value
```
Comment (harsha509) [inline javascript/node/selenium-webdriver/lib/bidiexecutor.js]: `return await f(this.#driver, command)`

File: javascript/node/selenium-webdriver/test/lib/port_bidi_classic_test.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+'use strict'
+
+const assert = require('assert')
+const { By, Browser } = require('../../index')
+const { Pages, ignore, suite } = require('../../lib/test')
+const firefox = require("../../firefox");
+
+suite(function (env) {
+  const browsers = (...args) => env.browsers(...args)
```
Comment (harsha509) [inline javascript/node/selenium-webdriver/test/lib/port_bidi_classic_test.js:26]: this can be removed.

**Review by harsha509 — COMMENTED** [review]
LGTM, only minor changes!

**Review by harsha509 — APPROVED** [review]

**Pull request closed without merging**